charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -200,6 +201,11 @@ spec:
 {{- end}}
       containers:
       - name: cassandra
+        volumeMounts:
+          - name: cassandra-config
+            mountPath: /etc/cassandra
+          - name: mcac-agent-config
```

Comment:
As mentioned in https://github.com/k8ssandra/k8ssandra/issues/292#issuecomment-774306539 we now have images that run as non-root. Have you tested the `mcac-agent-config` volume mount with new, non-root image?

Reply:
Tested them now, and it seems to work. Though I'm only testing with reaper enabled, not medusa (since I don't have any s3 or gcs buckets available).